src/lib/util/guid.ts

```diff
@@ -1,5 +1,5 @@
 export function guidGenerator() {
-  var S4 = function () {
+  let S4 = function () {
```

Comment:
`let` to `const` ?
Also should be moved outside guidGenerator no?
And `function () {` to `() => {`

Reply:
Not in scope to fix everything in this PR